app/src/androidTest/java/org/systers/mentorship/SignUpActivityTest.kt

```diff
@@ -225,10 +217,10 @@ class SignUpActivityTest {
     @Test
     fun whenUsernameIsAlreadyRegistered() {
         findEditTextInTextInputLayout(R.id.tiName).perform(typeText("name"), closeSoftKeyboard())
-        findEditTextInTextInputLayout(R.id.tiUsername).perform(typeText("Divyansh"), closeSoftKeyboard())
-        findEditTextInTextInputLayout(R.id.tiEmail).perform(typeText("justdvnsh2208@gmail.com"), closeSoftKeyboard())
-        findEditTextInTextInputLayout(R.id.tiPassword).perform(typeText("Divyansh@2001"), closeSoftKeyboard())
-        findEditTextInTextInputLayout(R.id.tiConfirmPassword).perform(typeText("Divyansh@2001"), closeSoftKeyboard())
+        findEditTextInTextInputLayout(R.id.tiUsername).perform(typeText("isabel"), closeSoftKeyboard())
```

Comment:
@isabelcosta sorry I couldn't really think of any other username so I used your, if we do end up creating a user solely for testing we can update it here.

Reply:
username is fine ;)